packit_service/worker/build/copr_build.py

```diff
@@ -185,7 +192,138 @@ def monitor_not_submitted_copr_builds(self, number_of_builds: int, reason: str):
                 reason=reason
             ).observe(time)
 
+    def get_packit_copr_download_urls(self) -> List[str]:
+        """
+        Get packit package download urls for latest succeeded build in configured project
+        (e.g. for production packit/packit-stable)
+        e.g .https://download.copr.fedorainfracloud.org/results/packit/
+        packit-dev/fedora-35-x86_64/02996411-packit/
+        python3-packit-0.42.1.dev8+g5f7e0b0-1.20211201073310374840.main.8.g5f7e0b0.fc35.noarch.rpm
+
+        Returns: list of urls
+        """
+        try:
+            latest_successful_build_id = (
+                self.api.copr_helper.copr_client.package_proxy.get(
+                    ownername="packit",
+                    projectname=self.service_config.packit_copr_project_name,
```

Comment:
There is also `deployment` (of type [Deployment](https://github.com/packit/packit-service/blob/main/packit_service/config.py#L35)) in the config.
Could that be used instead of adding another key?
```suggestion
                    projectname="packit-stable" if self.service_config.deployment == Deployment.prod else "packit-dev",
```